core/app/models/workarea/order.rb

```diff
@@ -242,9 +242,8 @@ def place
     def add_item(attributes)
       quantity = attributes.fetch(:quantity, 1).to_i
       sku = attributes[:sku]
-      customizations = attributes[:customizations]
 
-      if existing_item = items.find_existing(sku, customizations)
+      if existing_item = items.find_existing(sku, attributes)
```

Comment:
Why not just pass the whole hash here?
```suggestion
      if existing_item = items.find_existing(attributes)
```

Reply:
I felt that changing the arity of the method signature here would be an unnecessary breaking change.